app/src/components/market/profile/view.tsx

```diff
@@ -140,6 +152,7 @@ const ViewWrapper = (props: Props) => {
             </ButtonAnchor>
           )}
           <ButtonAnchor href={`/#/${marketMakerAddress}/buy`}>Buy</ButtonAnchor>
+          {isQuestionFinalized && <Button onClick={onResolveCondition}>Resolve condition</Button>}
```

Comment:
Q: Do we need to check here if it is not already resolved?

Reply:
Since this is in the page that is shown when the condition is not resolved, it's not necessary to also check that.